sdf/opUnion.glsl

```diff
@@ -10,7 +10,14 @@ use: <float> opUnion( in <float|vec4> d1, in <float|vec4> d2 [, <float> smooth_f
 #define FNC_OPUNION
 
 float opUnion( float d1, float d2 ) { return min(d1, d2); }
-vec4  opUnion( vec4 d1, vec4 d2 ) { return (d1.a < d2.a) ? d1 : d2; }
+
+Material opUnion( Material d1, Material d2 ) {
```

Comment:
Maybe add:

#if defined(STR_MATERIAL)

in case people use this function without material definition?
or
add #include "material.glsl" at the top